### DISCUSSION TITLE
Build on various platform

### DISCUSSION THREAD

**Discussion by 25gabriel-cmyk** [aahnik/tgcf#461] [General]
Thanks aahnik
you made my day, this is powerful and super easy to deploy app.
even for a beginner like me
i succesful build this on several platform 
- Heroku
- DO
- Gitpod
- PythonAnywhere (fail, file quota exceeded, frea account)
- back4app
- AWS server

**Comment by iamyashg** [comment]
Could you also try on Replit?

**Reply by 25gabriel-cmyk** [reply]
done, check here 
https://tgcf--25gabriel-cmyk.repl.co/

iam still figuring out how github-replit workflow to make pull request
also i dont know how longer it will live, looks like my replit free account running out credit


![6](https://github.com/aahnik/tgcf/assets/134001939/026f890a-c096-4673-9f90-e19eb974cf8a)

**Reply by iamyashg** [reply]
problem is not with deploying, i have also deployed it but in my case TGCF is not forwarding any message. i am using session string (telethon).

could you check that in your case is it forwarding messages from source to destination? using session string (if yes then tell from where did you generated that session string)

**Reply by 25gabriel-cmyk** [reply]
gosh u reply on my discussion about build on various platform, i think ur problem doesn't about the deployment.
Its about TG session string right? 
I reply there 
https://github.com/aahnik/tgcf/discussions/463

**Reply by iamyashg** [reply]
Yes

**Comment by indexytesla** [comment]
> Thanks aahnik you made my day, this is powerful and super easy to deploy app. even for a beginner like me i succesful build this on several platform
> 
> * Heroku
> * DO
> * Gitpod
> * PythonAnywhere (fail, file quota exceeded, frea account)
> * back4app
> * AWS server

Hi, Could you please specify how to run on gitpod? I'm having a hard time figuring that out from the docs.

**Reply by 25gabriel-cmyk** [reply]
Could u please specify what exactly the problem? I think build this repo on GitPod is the most easy way, as simple as build on DO Docker. 99% GUI experience. Read the doc carefully

**Comment by KaleemUllah789** [comment]
> Thanks aahnik you made my day, this is powerful and super easy to deploy app. even for a beginner like me i succesful build this on several platform
> 
> * Heroku
> * DO
> * Gitpod
> * PythonAnywhere (fail, file quota exceeded, frea account)
> * back4app
> * AWS server

Can you try to run tgcf on "cyclic.sh" which is free Cloud for apps and bots. If you succeeded in depolying on cyclic.sh let me know the steps so i can also deploy on that as i don't know how to deploy.
https://www.cyclic.sh/

**Reply by 25gabriel-cmyk** [reply]
will try to figuring out
